framework/src/bundle/BundleManifest.cpp

```diff
@@ -143,34 +177,41 @@ bool BundleManifest::Contains(const std::string& key) const
 Any BundleManifest::GetValue(const std::string& key) const
 {
   auto iter = m_Headers.find(key);
-  if (iter != m_Headers.end()) {
+  if (m_Headers.cend() != iter)
+  {
```

Comment:
The formatting of the if statement does not follow convention.

Reply:
Done.